modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model/channel-drift-flux.i

```diff
@@ -243,6 +243,7 @@ velocity_interp_method = 'rc'
     linear_coef_name = 'Darcy_coefficient'
     outputs = 'out'
     output_properties = 'vel_slip_x'
+    ghost_layers = 5
```

Comment:
not sure I understand this change?

Reply:
you have it in your physics input. If the test actually ran in CIVET it would fail without this change

Reply:
ah yes
```
        # Needed for some reason
        ghost_layers = 5
```